.github/workflows/integration.yml

```diff
@@ -226,6 +226,15 @@ jobs:
         with:
           ref: ${{ env.REF }}
 
+      - name: Pull images
```

Comment:
What's the difference between this and adding `&` in the action?

Reply:
This pulls all images in parallel (for all actions). In actions you download only one image (for example `hep`)and then you run it, then you download next image *for example `next` this time) and then you run it. which might be significantly slower.